server/src/email/templates/special-use-common/application-submitted-confirmation.es6

```diff
@@ -5,7 +5,7 @@ module.exports = (application, defaultApplicationDetails, reviewTime) => {
 
   return {
     to: application.applicantInfoEmailAddress,
-    subject: `Your ${util.camelCaseToRegularForm(application.type)} Open Forest permit application has been submitted for review!`,
+    subject: `Your ${application.type === 'tempOutfitters' ? 'Temporary Outfitting and Guiding' : 'Non-Commercial Group Use'} permit application has been submitted for review!`,
```

Comment:
Per our conversation offline, is there any way we can keep this generic instead of restricting it to two types in a ternary, so it handles any application type that might get added in the future?